test/framework/src/main/java/org/elasticsearch/search/aggregations/bucket/AbstractNXYSignificanceHeuristicTestCase.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.search.aggregations.bucket;
+
+import org.elasticsearch.search.aggregations.bucket.terms.heuristic.SignificanceHeuristic;
+
+/**
+ * Abstract test case for testing significant term heuristics
+ */
+public abstract class AbstractNXYSignificanceHeuristicTestCase extends AbstractSignificanceHeuristicTestCase {
+
+    /**
+     * @return A random instance of the heuristic to test
+     */
+    @Override
+    protected SignificanceHeuristic getHeuristic() {
+        return getHeuristic(randomBoolean(), randomBoolean());
+    }
+
+    /**
+     * @param includeNegatives value for this test run
+     * @param backgroundIsSuperset value for this test run
```

Comment:
Again, I think a little more detail would be helpful.  Think about reading this as a new developer coming to this code.  Include negative what?  What does `backgroundIsSuperset` mean?  Superset of what?

Reply:
`backgroundIsSuperset` is that its the super set of the "subset". Otherwise they are treated as two disjoint sets. Will add some docs.